packages/@react-spectrum/s2/intl/ar-AE.json

```diff
@@ -12,7 +13,7 @@
   "label.(optional)": "(اختياري)",
   "label.(required)": "(مطلوب)",
   "menu.moreActions": "المزيد من الإجراءات",
-  "picker.placeholder": "حدد خيارًا...",
+  "picker.placeholder": "تحديد…",
   "slider.maximum": "أقصى",
```

Comment:
English string changed to shorter select, update translations to match (many locales)